consensus/dummy/dynamic_fee_window_test.go

```diff
@@ -0,0 +1,246 @@
+// (c) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package dummy
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/ava-labs/coreth/params"
+	"github.com/ethereum/go-ethereum/common/math"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDynamicFeeWindow_Add(t *testing.T) {
+	tests := []struct {
```

Comment:
supernit you could use `map[string]struct{...}` to have the subtest name as the map key - it formats nicely imo the test cases. The only drawback (or not) is that subtests run without order.

Reply:
I generally find it easier to debug / work with a test where the order is consistent... So I always use the struct version rather than the map approach... But If people feel strongly I can switch to this.

Reply:
I also prefer the consistent order of arrays here